tests/testthat/test-pkgdown.R

```diff
@@ -0,0 +1,60 @@
+test_that("use_pkgdown() requires a package", {
+  create_local_project()
+  expect_usethis_error(use_pkgdown(), "not an R package")
+})
+
+test_that("use_pkgdown() creates and ignores the promised file/dir", {
+  local_mock(`usethis:::is_installed` = function(pkg) TRUE)
+  create_local_package()
+  use_pkgdown()
+  expect_true(uses_pkgdown())
+  expect_true(is_build_ignored("^_pkgdown\\.yml$"))
+  expect_true(is_build_ignored("^docs$"))
+})
+
+# pkgdown helpers ----
+test_that("pkgdown helpers behave in the absence of pkgdown", {
+  local_mock(`usethis:::is_installed` = function(pkg) TRUE)
```

Comment:
Shouldn't that be `FALSE`?

Reply:
No, this is so that the test can be run even if pkgdown is not installed. You don't really need pkgdown to check the pkgdown helpers. We have a similar pattern in many tests, when we do setup related to package X, but package X is not in usethis's Imports (or, often, not even in Suggests).